src/platform/packages/shared/response-ops/alerts-table/schemas/alerts_table_configuration_schema.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { z } from '@kbn/zod';
+
+/**
+ * Configuration schema for overrides to the table configuration that can be persisted across sessions.
+ * This should include only the serializable properties that are user-configurable, all other props
+ * will be removed by parsing to avoid saving unnecessary or non-serializable objects.
+ */
+export const alertsTableConfigurationSchema = z.object({
```

Comment:
Motivation behind this new schema is described here: https://github.com/elastic/kibana/pull/234309#issue-3394139784